src/LiveQuery/QueryTools.js

```diff
@@ -90,6 +90,28 @@ function queryHash(query) {
 }
 
 /**
+ * contains -- Determines wether a constaint in for form of a list
```

Comment:
"whether" and "constraint" are misspelled.  I haven't read the code yet, but I really can't understand this comment.